### ISSUE TITLE
gpu mining option not really working?

### ISSUE THREAD

**Issue by DrekerboyStudio** [UnamSanctam/SilentXMRMiner#240]
Hi ive been testing this builder for some time and its amazing!!!
but now ive been testing the gpu miner option but it doesnt seem to work? 
it startsup and i injected it in notepad so I watch the taskmanager to monitor it. after some time you will see the gpu percentage go up to around 35% and then immediatly it stops and doesnt use the gpu at all after that.
ive tried this with NVIDIA card and with AMD card and on the AMD is the only one that does this because NVIDIA doesnt even do that it just never uses the gpu at all.
stealth is turned off, cpu mining is off, installation is off,
ive tried multiple drivers for it older versions and newer versions, on the newest version it gives error that the driver got a timeout. on the older  versions it doesnt give you the error but it will only use the gpu for that one second.
am I doing something wrong?
any tips maybe?

**Comment by UnamSanctam** [comment]
What algorithm/coin are you trying to mine? Also, for most algorithms it should only show between 0-5% GPU usage since it doesn't use any CUDA cores to do any work. What you should check to make sure that it's working is the VRAM and memory clock. If you use something like Process Hacker and double-click the mining process and then check the GPU tab and if it's using a few GB of VRAM (GPU Memory) then it's working. I also have an Ethash GPU miner as well depending on what you're planning to mine: https://github.com/UnamSanctam/SilentETHMiner/

**Comment by UnamSanctam** [comment]
Yes it should do that, it should only use a bit of GPU when it's initializing and then run at 0%. Check the GPU Memory with Process Hacker like how I said in my earlier reply and you should see if it's working or not.

**Comment by DrekerboyStudio** [comment]
I did, same thing the spike happens and then nothing.
even after days of testing multiple things and letting it run for hours straight it also never showed up in my mining pool, also not in the history that it even connected for a second. I made a version that only mines cpu with the exact same settings and that does show up in the mining pool.

**Comment by UnamSanctam** [comment]
The spike happens in the Process Hacker GPU memory? Also, which GPU do you have?

**Comment by DrekerboyStudio** [comment]
Yes the GPU spike of 35% happens also in Process Hacker and after that nothing uses the GPU anymore. after that spike the GPU wont be used anymore but notepad will start using 75% of RAM i dont know why but this will not go down.
its an older model AMD XFX R7900 GHOST, its just for testing. I also tried with multiple NVIDIA cards newer ones also but there it doesnt seem to even use any GPU at all and also I let it ran with NVIDIA GPU for hours for each single card. 
it only seems to even do anything with the AMD card but only that spike and then nothing.

**Comment by DrekerboyStudio** [comment]
I know it is not profitable, it is just a friend wanted to mine xmr on big scale in his warehouse and asked me to test it out for him thats why I know I should be using this builder for big scale operation but I had this one and wanted to try first if it even worked.
![image](https://user-images.githubusercontent.com/55652473/117459862-28187980-af4c-11eb-98d1-1552f88c0358.png)

**Comment by DrekerboyStudio** [comment]
nope

**Comment by DrekerboyStudio** [comment]
If im the onlyone that has this problem its okay

**Comment by UnamSanctam** [comment]
Could you try and run this and see if the same happens?
[TESTGPU.zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6442027/TESTGPU.zip)

**Comment by DrekerboyStudio** [comment]
same thing happened its probably just me that has this problem, its okay still love your work!
![image](https://user-images.githubusercontent.com/55652473/117461193-93168000-af4d-11eb-9021-c2947c4c7450.png)

**Comment by DrekerboyStudio** [comment]
Should I close the issue for you?

**Comment by UnamSanctam** [comment]
Ok, sure you can do that. Also just as a final thing, I recommend mining Monero on the CPU and Ethereum on the GPU at the same time for maximum profit on systems that have both.

**Comment by DrekerboyStudio** [comment]
Thanks for the tip, I will be testing that!
